modules/transformation.py

```diff
@@ -85,6 +85,7 @@ def __init__(
             "create_disposition": BigQueryDisposition.CREATE_IF_NEEDED,
             "write_disposition": BigQueryDisposition.WRITE_APPEND,
             "additional_bq_parameters": {
+                "maxBadRecords": 10,
```

Comment:
This is the actual fix. The rest just cleans up errors that do not cause a full failure but do cause errors.

Still can't see what exactly failed and why.